uni/lib/model/entities/lecture.dart

```diff
@@ -1,6 +1,25 @@
+import 'package:json_annotation/json_annotation.dart';
 import 'package:logger/logger.dart';
 
+part '../../generated/model/entities/lecture.g.dart';
+
+class DateTimeConverter extends JsonConverter<DateTime, String> {
+  const DateTimeConverter();
+
+  @override
+  DateTime fromJson(String json) {
+    return DateTime.parse(json);
+  }
+
+  @override
+  String toJson(DateTime object) {
+    return object.toIso8601String();
+  }
+}
+
```

Comment:
I think we should not have repeated converters, especially if it's a DateTime converter.